scripts/freesurfer/recon_all.py

```diff
@@ -109,4 +107,4 @@ def main(*, n_jobs: int = 1) -> None:
 
 
 if __name__ == '__main__':
-    fire.Fire(main)
+    main()
```

Comment:
Actually I think this can be removed now – the "proper" way to run this step is through `run.py`

Reply:
Actually I was mistaken here, `run.py` currently runs all processing steps as scripts, so we do need this bit